ohttp/src/hpke.rs

```diff
@@ -27,6 +27,15 @@ macro_rules! convert_enum {
     }
 }
 
+#[cfg(feature = "rust-hpke-pq")]
+convert_enum! {
+pub enum Kem {
+    X25519Sha256 = 32,
+    X25519Kyber768Draft00 = 48,
```

Comment:
This should be possible...

```suggestion
    #[cfg(feature = "pq")]
    X25519Kyber768Draft00 = 48,
```

with 
```diff
diff --git a/ohttp/src/hpke.rs b/ohttp/src/hpke.rs
index 7bca571..3e5eb79 100644
--- a/ohttp/src/hpke.rs
+++ b/ohttp/src/hpke.rs
@@ -13,7 +13,9 @@ macro_rules! convert_enum {

             fn try_from(v: u16) -> Result<Self, Self::Error> {
                 match v {
-                    $(x if x == $name::$vname as u16 => Ok($name::$vname),)*
+                    $($(#[$vmeta])*
+                      x if x == $name::$vname as u16
+                      => Ok($name::$vname),)*
                     _ => Err(crate::Error::Unsupported),
                 }
             }
```

(which corrects an error on my part)